src/utils/config.ts

```diff
@@ -6,10 +6,14 @@ type configType = {
   SCHEMA: string
   STRIPE_SECRET_KEY: string
   STRIPE_WEBHOOK_SECRET: string
```

Comment:
Looks like this has been removed from `.env.sample`

Reply:
Tests were failing with or without it so I wasn’t sure whether this should be removed.

I could remove it if I can get the tests to pass.

Reply:
Also: 
Users could leave empty the STRIPE_WEBHOOK_URL property and fill this one to opt-out from the automatic creation of the hook.

Reply:
Maybe i should clarify this in the documentation

Reply:
The tests pass on my local, but the need a bit of set up:

- create a database on Supabase
- grab the Database URL from the settings
- fill out the `.env` file:

```
NODE_ENV=production
STRIPE_SECRET_KEY=sk_test_XXXX
STRIPE_WEBHOOK_SECRET=whsec_XXXX
SCHEMA=stripe
DATABASE_URL=postgres://postgres:XXXX@db.XXX.supabase.co:5432/postgres?sslmode=disable&search_path=stripe
```

- Run `dbmate up` (will create the stripe tables)
- Run  `npm run test`

I guess I should add a local postgres with docker to make this process a bit simpler

Reply:
Sent a new commit

Reply:
Yes that would make more sense

Reply:
OK so now that I read through the PR, is there a way that we can mock Stripe so we don't need an account? Previously I was just testing again local copies of Stripe events, and now it looks like it will need a valid `STRIPE_SECRET` (correct me if I'm wrong

Reply:
yes, we can mock stripe if we do not use it directly. we could create helper functions that do the thing we want and we can have a mock version of them instead of a mocked stripe library which is way too complex.

Reply:
There are ways we could avoid auto-creation of webhook by not supplying the webhook URL. hence the creation will be skipped.

Reply:
@kiwicopple  What do you think?